### ISSUE TITLE
训练UserWarning

### ISSUE THREAD

**Issue by Fourpk** [xy-guo/MVSNet_pytorch#56]
运行./train.sh，出现以下信息：
Epoch 0:
/root/autodl-tmp/tutorial-env/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:139: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
/root/autodl-tmp/tutorial-env/lib/python3.8/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3483.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
/root/autodl-tmp/tutorial-env/lib/python3.8/site-packages/torch/nn/functional.py:4236: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn(
/root/autodl-tmp/tutorial-env/lib/python3.8/site-packages/torch/nn/_reduction.py:42: UserWarning: size_average and reduce args will be deprecated, please use reduction='mean' instead.
  warnings.warn(warning.format(ret))
Epoch 0/16, Iter 0/6774, train loss = 63.717, time = 13.752
Epoch 0/16, Iter 1/6774, train loss = 75.159, time = 0.626
Epoch 0/16, Iter 2/6774, train loss = 71.151, time = 0.626
刚开始的训练损失特别大，请问这些警告会影响训练效果吗
